docs/howto/kubernetes/gardener/hibernate-shoot-cluster.md

```diff
@@ -21,22 +21,27 @@ Click anywhere on its row for a detailed view of the various cluster characteris
 Bring up the *Status* tab and, in the *Constraints* section, check whether hibernation is possible.
 In the example below we have a shoot cluster based on Kubernetes 1.28, the latest supported at the time of this writing, and hibernation is indeed possible.
 
-![{{k8s_management_service}} cluster status](assets/garhiber-01.png)
+![{{k8s_management_service}} cluster status](assets/shoot-hiber-01.png)
 
 To go ahead and actually hibernate the cluster, click the orange :material-dots-horizontal-circle: icon at the right-hand side of the cluster row.
 From the pop-up menu that appears, select *Hibernate Cluster*.
 
-![Hibernate cluster](assets/garhiber-02.png)
+![Hibernate cluster](assets/shoot-hiber-02.png)
 
 A big pop-up window labeled *About to hibernate a gardener shoot* appears, asking you to confirm the action.
 Click the red button labeled *Yes, Hibernate*.
 
-![Confirm cluster hibernation](assets/garhiber-03.png)
+![Confirm cluster hibernation](assets/shoot-hiber-03.png)
+
+The cluster will begin hibernating.
+The progress of the whole process will be visible by the animated icon, at the left-hand side of the cluster row.
```

Comment:
"The progress of the whole process" sounds weirdly awkward to me because of the near-repetition. Also, the "will be visible" construction feels oddly detached.

```suggestion
You will be able to monitor progress via the animated icon, at the left-hand side of the cluster row.
```

Reply:
Yes and yes :|

Reply:
For the record, I don't know if "monitor the progress" or "monitor the process" is better — what do you think?

Reply:
Hm, I do not like either, to be honest. How about "The animated icon at the left-hand side of the cluster row marks the progress"?

Reply:
Well, I am amending anyway :)

Reply:
That works too.